core/dbt/include/global_project/macros/materializations/models/table/create_table_as.sql

```diff
@@ -25,11 +25,26 @@
 
```

Comment:
We need the get_select_subquery in order to put the columns in the right order.  We removed validating the order because we added the subquery. As far as splitting out the macro into two, probably @jtcohen6 and @MichelleArk should weigh in on that.